Sources/AnyLint/Checkers/TemplateChecker.swift

```diff
@@ -0,0 +1,27 @@
+import Foundation
+
+/// The source of the subchecks to run.
+public enum CheckSource {
+    /// The device-local source, requiring a path String.
+    case local(String)
+
+    /// A remote public URL source, requiring the URL string.
+    case remote(String)
+
+    /// Predefined and officially supported community projects providing different variants of checks.
+    case community(String, variant: String)
+}
+
+struct TemplateChecker {
+    let source: CheckSource
+    let runOnly: [String]?
+    let exclude: [String]?
+    let options: [String: Codable]?
+}
+
+extension TemplateChecker: Checker {
+    func performCheck() throws -> [CheckInfo : [Violation]] {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Colons should be next to the identifier when specifying a type and next to the key in dictionary literals.](https://app.codacy.com/gh/Flinesoft/AnyLint/pullRequest?prid=5691372)